frontend-instructor/src/components/dashboard/Dashboard.jsx

```
@@ -1,15 +1,84 @@
 
-export default Dashboard;
+              <div className="ml-4 flex-1">
+                <h3 className="text-lg font-semibold">{course.title}</h3>
+                <p className="text-gray-600">Instructor: {course.instructor?.name || "Unknown"}</p>
+                <p className="text-yellow-500">Rating: {course.rating ?? "N/A"} ⭐</p>
+                <p className="text-lg font-bold text-blue-500">৳ {course.price}</p>
+
+                <div className="mt-2 flex gap-3">
+                  <Link to={`/courses/${course.id}`}>
+                    <button className="px-4 bg-blue-500 text-white py-2 rounded-lg hover:bg-blue-600 transition">
+                      View Details
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)